README.md

```diff
@@ -387,6 +393,10 @@ buildctl build ... \
   --import-cache type=gha
 ```
 
+Note that layers are saved to GitHub Actions cache. This cache currently has a [size limit of 10GB](https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy) that is shared accross different caches in the repo. If you exceed this limit, GitHub will save your cache but will begin evicting caches until the total size is less than 10 GB. Recycling caches too often can result in slower runtimes overall.
+
+Following the scope scheme of `actions/cache`, caches are scoped by branch, with the default branch being available to every branch. E.g. a pull request to the default branch can load cache generated by the default branch but it can not overwrite that cache. Caches generated in pull requests are only used when the same pull request needs to be rebuilt. Caches generated for the default branch are used in next default branch builds and for all pull requests.
```

Comment:
"Following the scope scheme of `actions/cache`," -> Similarly to using [`actions/cache`](https://github.com/actions/cache),

Reply:
https://github.com/moby/buildkit/pull/2489/commits/2ba2d270fa871dbfa70097fff9c8693e655d05a9